google-cloud-examples/src/main/java/com/google/cloud/examples/bigtable/HelloWorld.java

```diff
@@ -0,0 +1,183 @@
+    BigtableDataSettings settings =
+        BigtableDataSettings.newBuilder()
+            .setInstanceName(InstanceName.of(GCLOUD_PROJECT_ID, INSTANCE_ID))
+            .build();
+
+    // Create bigtable data client
+    BigtableDataClient dataClient = BigtableDataClient.create(settings);
+
+    // Create the settings to configure a bigtable admin client
+    BigtableTableAdminSettings adminSettings =
+        BigtableTableAdminSettings.newBuilder()
+            .setInstanceName(
+                com.google.bigtable.admin.v2.InstanceName.of(GCLOUD_PROJECT_ID, INSTANCE_ID))
+            .build();
+
+    // Create bigtable admin client
+    BigtableTableAdminClient adminClient = BigtableTableAdminClient.create(adminSettings);
+    // [END connecting_to_bigtable]
+
+    try {
+      // Check if table exists, create table if does not exist
+      createTable(adminClient, TABLE_ID, COLUMN_FAMILY_ID);
+
+      // Write to table
+      writeToTable(dataClient, TABLE_ID, ROW_KEY_PREFIX, COLUMN_FAMILY_ID, COLUMN_QUALIFIER);
+
+      // Read single row from the table
+      readSingleRow(dataClient, TABLE_ID, ROW_KEY_PREFIX);
+
+      // Read entire table
+      readTable(dataClient, TABLE_ID);
+
+      // Delete table
+      deleteTable(adminClient, TABLE_ID);
+    } catch (Exception e) {
+      System.out.println("Exception while running HelloWorld: " + e.getMessage());
+    } finally {
+      dataClient.close();
+      adminClient.close();
+    }
+  }
+
+  public static Table createTable(
+      BigtableTableAdminClient adminClient, String TABLE_ID, String COLUMN_FAMILY_ID) {
+    // [START creating_a_table]
+    if (adminClient.exists(TABLE_ID)) {
+      System.out.println("Deleting existing table");
+      adminClient.deleteTable(TABLE_ID);
+    }
+    CreateTableRequest createTableRequest =
+        CreateTableRequest.of(TABLE_ID).addFamily(COLUMN_FAMILY_ID);
+    System.out.println("Creating table: " + TABLE_ID);
+    Table table = adminClient.createTable(createTableRequest);
+    return table;
+    // [END creating_a_table]
+  }
+
+  public static RowMutation writeToTable(
+      BigtableDataClient dataClient,
+      String TABLE_ID,
+      String ROW_KEY_PREFIX,
+      String COLUMN_FAMILY_ID,
+      String COLUMN_QUALIFIER) {
+    // [START writing_rows]
+    RowMutation mutation = null;
+    try {
+      System.out.println("Write some greetings to the table:");
+      String[] greetings = {"Hello World!", "Hello Bigtable!", "Hello Java!"};
+      for (int i = 0; i < greetings.length; i++) {
+        RowMutation rowMutation = RowMutation.create(TABLE_ID, ROW_KEY_PREFIX + i);
+        mutation = rowMutation.setCell(COLUMN_FAMILY_ID, COLUMN_QUALIFIER, greetings[i]);
+        dataClient.mutateRow(rowMutation);
+        System.out.println(greetings[i]);
+      }
+    } catch (Exception e) {
+      System.out.println("Exception while writing to table: " + e.getMessage());
+    }
+    return mutation;
```

Comment:
I did this for test purposes. Do you prefer I do it a different way? Also, as per your comment below, once the methods are made non-static, I can't call them in my test class so I won't be able to test each specific method. Do you want me to just test that main doesn't crash? If so, I can take out all the return statements.

Reply:
Your test cases can instantiate the class and then call the individual methods like before. (just passing the parameters in the constructor instead of the method)

Reply:
Is it okay to leave the return statements then?

Reply:
I'd prefer if the test read the rows from bigtable instead